### DISCUSSION TITLE
Add fontSize prop to Separator

### DISCUSSION THREAD

**Discussion by mturoci** [h2oai/wave#651] [Ideas]
via @vopani

**Comment by lo5** [comment]
One of the drawbacks of allowing font size changes to the separator is that doing so would hamper consistency of look and feel. 

That said, this ask might be because the `separator` does a better job of visually separating a form's sections (compared to `ui.text_*()`) because of the existence of the horizontal line across the text. If so, instead of allowing arbitrary font sizes, we should probably use preset heading levels (see `size` argument to `ui.text()`).

cc @vopani
